companion/src/simulation/telemetrysimu.cpp

```diff
@@ -659,7 +659,7 @@ void TelemetrySimulator::LogPlaybackController::calcLogFrequency()
     QDateTime logTime = parseTransmittterTimestamp(csvRecords[i]);
     // ugh - no timespan in this Qt version
     double timeDiff = (logTime.toMSecsSinceEpoch() - lastTime.toMSecsSinceEpoch()) / 1000.0;
-    if ((timeDiff > 0.1) && (timeDiff < logFrequency)) {
+    if ((timeDiff >= 0.1) && (timeDiff < logFrequency)) {
```

Comment:
Actually I would change this to `timeDiff > 0.09` because equal operator for floating point is basically useless.